assets/components/stripePopUpButton/stripePopUpButton.jsx

```diff
@@ -37,7 +38,7 @@ const StripePopUpButton = (props: PropTypes) => {
     <button
       className="component-stripe-pop-up-button"
       onClick={stripeClick}
-    >Contribute with debit/credit card</button>
+    >Contribute with debit/credit card <Svg svgName="credit-card" /></button>
```

Comment:
After an offline conversation, we decided that we probably need to rename the SVG module to avoid a collision with the `svg` tag. Good point @rupertbates 👍